plugins/inara.py

```diff
@@ -584,11 +726,12 @@ def journal_entry(cmdr, is_beta, system, station, entry, state):
             if this.storedmodules != modules:
                 # Only send on change
                 this.storedmodules = modules
-                this.events = [x for x in this.events if x['eventName'] != 'setCommanderStorageModules']	# Remove any unsent
+                this.events = [x for x in this.events if x['eventName']
+                               != 'setCommanderStorageModules']  # Remove any unsent
```

Comment:
Breaking the line mid conditional feels *so* wrong.

1) Move the comment to its own line above?
2) Split at the 'if ' otherwise/if you still need to ?

Reply:
oh I didnt commit this, yeah I had done that. I blame auto-formatters